tests/integration/pilot/gateway_test.go

```diff
@@ -563,3 +563,36 @@ func StatusGatewayTest(t framework.TestContext) {
 	// It should be added back
 	retry.UntilSuccessOrFail(t, check)
 }
+
+// Verify that the envoy readiness probes are reachable at
+// https://GatewayPodIP:15021/healthz/ready . This is being explicitly done
+// to make sure, in dual-stack scenarios both v4 and v6 probes are reachable.
+func TestGatewayReadinessProbes(t *testing.T) {
```

Comment:
you can call the gw Service, 15021 is exposed in the Service.

I think?

Reply:
https://github.com/istio/istio/pull/44024 is not yet merged, and that is another bug that gateway svc is not deployed as a dual-stack service.